docs/en/stack/ml/nlp/ml-nlp-inference.asciidoc

```diff
@@ -111,6 +122,42 @@ Hungarian.
 .. If everything looks correct, close the panel, and click **Create
 pipeline**. The pipeline is now ready for use.
 
+////
```

Comment:
FYI I've added the failure processor into the "Add an inference processor..." step, rather than at the end. I've commented out the API example for now, since we don't currently have those in this page. However, I will circle back in a later PR to add a full API example since @sophiec20 asked for this too.